po/R-fr.po

```diff
@@ -0,0 +1,645 @@
+msgid "No character column found from the input data."
+msgstr "Aucune colonne de type charactère n'a été identifiée."
+
+#: clean_data_helpers.R:91
+msgid ""
+"Found {.cls numeric} values that can also be of type {.cls Date} in: {.field "
+"{ambiguous_cols}}."
+msgstr ""
+"Des valeures {.cls numériques} pouvant aussi être de type {.cls Date} ont "
+"été identifiées dans: {.field {ambiguous_cols}}."
+
+#: clean_data_helpers.R:92 standardize_date.R:185
+msgid ""
+"They can be converted into {.cls Date} using: {.code lubridate::as_date(x, "
+"origin = as.Date(\"1900-01-01\"))}"
+msgstr ""
+"Veuillez utiliser {.code lubridate::as_date(x, origin = as."
+"Date(\"1900-01-01\"))} pour les convertir en {.cls Date}"
+
+#: clean_data_helpers.R:93 standardize_date.R:186
+msgid ""
+"where {.val x} represents here the vector of values from these columns ({."
+"code data$target_column})."
+msgstr ""
+"où {.val x} représente le vecteur des valeurs issues des colonnes cibles ({."
+"code data$target_column})."
+
+#: column_name_standardization.R:56
+msgid "Unrecognised column names specified in {.emph rename}."
+msgstr ""
+"Certains noms de colonnes fournis dans {.emph rename} ne sont pas reconnus."
+
+#: column_name_standardization.R:57
+msgid "Make sure that the columns to be renamed are part of the input data."
+msgstr ""
+"Assurez-vous que toutes les colonnes à renommer fassent parti des données "
+"d'entrée."
+
+#: column_name_standardization.R:58
+msgid ""
+"To rename columns, use: {.emph rename = c(new_name1 = 'old_name1', new_name2 "
+"= 'old_name2')}."
+msgstr ""
+"Les noms de colonnes peuvent être renommer comme suit: {.emph rename = "
+"c(nouveau_nom1 = 'ancien_nom1', nouveau_nom2 = 'ancien_nom2')}."
+
+#: column_name_standardization.R:66
+msgid "Provided replace column names already exist."
+msgstr "Les noms utilisés pour renommer certaines colonnes existent déjà."
+
+#: column_name_standardization.R:67
+msgid "All new names must be different from existing column names."
+msgstr "Tout nouveau nom doit être différent des noms qui existent déjà."
+
+#: column_name_standardization.R:68
+msgid ""
+"You must use a different name for the following columns: {.field "
+"{toString(existing_cols)}}."
+msgstr ""
+"Vous devez fournir des noms différents pour les colonnes suivantes: {.field "
+"{toString(existing_cols)}}."
+
+#: convert_to_numeric.R:40
+msgid "Automatic detection of columns to convert into numeric failed."
+msgstr "Impossible de détecter automatiquement les colonnes à convertir."
+
+#: convert_to_numeric.R:41
+msgid "No character column with numeric values found by {.fn scan_data}."
+msgstr ""
+"Aucune colonne de type charactère contenant des chiffres n'a été identifiée "
+"par {.fn scan_data}."
+
+#: convert_to_numeric.R:42 convert_to_numeric.R:64
+msgid ""
+"Please specify names of the columns to convert into numeric using {.emph "
+"target_columns}."
+msgstr ""
+"Veuillez entrer les noms des colonnes à convertir en chiffre à travers "
```

Comment:
In other occurrences, you have translated "numeric" to "numérique". It is probably quite important to ensure you are consistent on this kind of technical terms.

I would check what base R is doing and reuse the same translation as them.

Once you have settle on a term, could you please add it to a draft glossary we can ultimately add to an eventual translation guide for Epiverse?